lib/patches/rake-compiler/rake/baseextensiontask.rb

```diff
@@ -0,0 +1,17 @@
+Truffle::Patching.require_original __FILE__
+
+module Truffle::Patching::Rake::BaseExtensionTaskOverrideBinaryExt
```

Comment:
I don't think this will work without a separate
`module Truffle::Patching::Rake; end`.
Instead, it will just define `Rake::BaseExtensionTaskOverrideBinaryExt`.

So either make sure `Truffle::Patching::Rake` is defined or I think it's just simpler to have the prefix prepended so we avoid an extra module + singleton class + extra code.
```ruby
module Truffle::Patching::RakeBaseExtensionTaskOverrideBinaryExt
```

Reply:
Fixed, thanks.